### ISSUE TITLE
Server down?

### ISSUE THREAD

**Issue by FranklinYu** [valery1707/javadoc-badge#6]
Currently all three examples doesn’t work; browser cannot connect to host `javadoc-emblem.rhcloud.com`.

**Comment by valery1707** [comment]
Yes, as said in https://github.com/valery1707/javadoc-badge/issues/2#issuecomment-344905098 project is closed
